meta-genivi-dev/meta-genivi-dev/recipes-extended/cannelloni/cannelloni.bb

```diff
@@ -0,0 +1,41 @@
+SUMMARY = "Add cannelloni to GDP and lunch it at boot time"
+SECTION = "cannelloni"
+LICENSE = "MIT"
```

Comment:
Please re-check the LICENSE, It should be [GPLv2](https://github.com/mguentner/cannelloni/blob/master/gpl-2.0.txt)